nmigen/vendor/intel.py

```diff
@@ -371,3 +371,28 @@ def get_diff_input_output(self, pin, p_port, n_port, attrs, invert):
             i_oe=self._get_oereg(m, pin),
         )
         return m
+
+    def get_ff_sync(self, ff_sync):
+        m = Module()
+        m.submodules += Instance("altera_std_synchronizer",
+            p_depth=ff_sync._stages, 
+            i_clk=ClockSignal(ff_sync._o_domain),
+            i_reset_n=1,
+            i_din=ff_sync.i,
+            o_dout=ff_sync.o,
+        )
+        return m
+
+    def get_reset_sync(self, reset_sync):
+        m = Module()
+        reset = Signal()
+        reset_clk = ClockSignal(reset_sync._domain)
```

Comment:
I think you can just use `ClockSignal` and `ResetSignal` directly in instance ports.